sass-lint.yml

```diff
@@ -0,0 +1,55 @@
+#########################
+## Starter config file
+#########################
+# Linter Options
```

Comment:
I believe we would need to coordinate some additional items with devops but this story was more to get it formally into npm (versioning etc) and to just get a linter in to start customizing rules. Does this impact current workflow for you all?  Definitely feel free to update the rules as desired.